.github/workflows/ci.yml

```
@@ -87,9 +87,17 @@ jobs:
         cache: true
 
     - name: Run Gosec Security Scanner
+      # The recommended way to run the gosec scanner is to use -no-fail
+      # https://github.com/securego/gosec#integrating-with-code-scanning
+      # The problem is that this can hide some errors (such as golang version mismatch)
+      # that are not caught by the following steps.
+      env:
+        # make sure that we use the golang version required by the project
+        GOTOOLCHAIN=auto
```

actionlint (1.7.12)

[error] expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)